packaging/install/drain-cleaner/kubernetes/040-Secret.yaml

```diff
@@ -0,0 +1,11 @@
+  tls.key: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2UUlCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktjd2dnU2pBZ0VBQW9JQkFRQzJCSFc1TkhaYVlIM0cKMHZLMEV6dWFQTkR2cUhKT1dadmNUNFhWVW5KaFN6T1d4MTFuZnlzNUVYRHFiYmJWck9aVzNvMmgwUEVhM0YweQpzQ21tem1qMUJ3dksrUnhXM1diSUxNM1FFQUNPMkI1ZDMyMTB0a1VIZk9UT2ZESS9CS2QwK3JvOUs5NUVIMkU3Cm95VnhhK2NhaTgvSFJpRGI1ZVFFQXUxclgwOHdOeW1GWkpVeGtHN3VNNmRydjZiWEhSYXdDRUpBNVZicHdZZlkKbmJQbm10UDBLWHViUU1ZVFF1bnphekJ6czBJUFd1TGF2dGE4VlZ3RkJyRWZ3OFVCUjZDT2Y2VUgyRGNyTENTcwo2OHFWdXYzamg0NkphT2lBV2dodUxIZ2RFTDFsYmNmYUlrcldZLzRvMmFCK2tWQ25LS1V2THBlQnhNSGVRcitPCmFpRlgrbXYxQWdNQkFBRUNnZ0VCQUlyMGZLc1hVZ05OQnJoeThyWm11R25YWStxU1BqRnlWM3FwZXhLUURxSVkKTnhvMHVTbjUrNVZscXo1TTQvZmNOenk2U29YdWJTS0VlT0hwZXVMTlVYTStIZEttdXV2VnVsRzNZcWdrajBoOQoxVEQxRVNxNHU1b3J6c0dVTHdmTkZjcElYMFd6NThzNTZaRU55OWY2K0VOYjJSUFloZDFhYytoY0U4U1V6bStQClk0c0NISEppR1g0Y3dLRUVoUnlCVWZqYm95MHJsTkVDUm5XVWFvWThYdjIwaW91dVpaSkkySklCaHNnRUpwSm8KZHNjYnNZOUJ6c3BBZnI2em02d0g2K2thM2lKMzhBb3gzeE9yY3AwczhHNUNjNDBYb2JyK1h3QW9ITjRRK2M3VApvZ3NqTU1DKzRrK2k4V0dQU1FkVlR2eWk5ay9ITUlYQXdUK0pWVFRhT1VFQ2dZRUEzRVlBUFdJYnJlV2QxS2UyCndEWWxzazFadDdYamJwSkpqVlJ3VjgyaUFRVXM5NWxyd0xkS3c3U1d2RHVXcnFVamMzck0wYlRodVhsRDBRcUcKVGwvczg2TFJBWGtqNFh0eFVaL2ZIS2R4bnNIU1JTa3JhVExSVWJ5ZDErMUxHbHJqMVkwaVJxbkhRVU1Wa1E1Nwo1Y3VTTks1NEN1VWQveVg4RGJpVzI4Ri9CQk1DZ1lFQTA0b0d1dTNXSFovdExyYm8yL3NSMTlrOUF0ZmpNbk1GCmtrSTMxcHBPR204c1NMdmZGWkRJTTRqSlcxMU9NZUNGS1lzOW52ZE9laCtlSnhENGowWTNtSExQbURFRVRuaWYKZzl0TlVwcXp1dGtUcTdPR002VE43Q2toR2lOQjN5SU1SbmVMWUVGVG5VM2Y0OXg1RS9aZDZ5Rm93b3gwVkNIZworcklvb3E0NkFOY0NnWUJYSFh3MHlwdEYwQnoyb0ltL1JPcnR0SGU2Rno1bVdZYXJWV3pFM1Blb1RhSzJIY0FtCnpKYk9CWnJqd1BGY0NzV1ZnMGlEMjNTMlJNWlVZMFF3dFBGaWZjOEpOUjdDcTI1cXFMNjFkMmVSTVBoM0Q4d3kKWTcyekJOU3NOQlhTeTRWMXpNWjBpZXJhM2F3bTNVS1U1RWpqZXVqUTdrSkp5WDBBOURMOW9HNGFid0tCZ0JtNApEbGxyMFR5STFTMTR6NWJ2VDV3T3JYeExBelIvQjQ1Z3ZxazQyWVlkUGlVWnNtWE0xSXB2ZHd2RXJGMGtEd3llCkFUL3lrWGdIb0JMZjNWaU9FUFJZQVNWZjBFWTZKQ3JEM3dMTURFNExWbmpwdXNkTW9qeTBRM0lLRjFncHE1UmwKdEd4ZmhPcjdMTjFuVU5FOGd1NnpnU2ttS242UUgrcGEvdFhJbVFUQkFvR0FPcCt0Rnl5ZG16RFJVZWwxMlhEUApFaFB3bmxDdzNSVjhrUUpjRk00eXhCbC9wQmp5K29vNU1NSlZyOFpQKzc1d1FSWkU5d21PZjk2MUVtV2N0VFZMClNVaDl4R3ljN2diY01JRlVQdXlnenVJNkNjWVhsRCtiR1VpUngrb29hRkNKL0pGVkh6bnRHc0RZY0J4TW13OUcKRFY0U2ZtZnlMeTRyRlFYdSsyRGVvY0k9Ci0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0K
```

Comment:
Are these hard coded cert+key?

Reply:
Yes, this is the prepared key for the pre-defined namespace. If you would not have it there, everyone would need to first generate the key and update all the resources.